### PR TITLE
Remove exception details from warn error message

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Remove exception from warning - removes clutter in openshift logs

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

### CODE REVIEW

File: src/main/java/ca/bc/gov/iamp/bcparis/filter/LoggingFilter.java [hunk 1]
```diff
@@ -31,28 +31,24 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
...
 
             MDC.put("messageId", messageId);
             MDC.put("correlationId", correlationId);
             MDC.put("mdcData", String.format("[msgId:%s, corlId:%s]", messageId, correlationId));
 
             chain.doFilter(copiedRequest, response);
+        } catch (Exception ex) {
+            log.warn("Failed to parse request body at logging filter");
         } finally {
             MDC.clear();
```
Comment (alexjoybc) [inline src/main/java/ca/bc/gov/iamp/bcparis/filter/LoggingFilter.java]: remove by keys

File: src/main/java/ca/bc/gov/iamp/bcparis/filter/LoggingFilter.java [hunk 2]
```diff
@@ -31,28 +31,24 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
...
             }
-        } catch (Exception ex) {
-            log.warn("Failed to parse request body at logging filter", ex);
-        }
-
-
-        // Logging
-        try {
 
             MDC.put("messageId", messageId);
```
Comment (alexjoybc) [inline src/main/java/ca/bc/gov/iamp/bcparis/filter/LoggingFilter.java]: put it in the if and use const

**Review by alexjoybc — CHANGES_REQUESTED** [review]

File: src/main/java/ca/bc/gov/iamp/bcparis/filter/LoggingFilter.java [hunk 3]
```diff
@@ -21,38 +21,32 @@
...
 
         // Message parsing
         try {
+
             l7message = new ObjectMapper().readValue(copiedRequest.getRequestBody(), Layer7Message.class);
             if (l7message != null) {
-                messageId = l7message.getEnvelope().getMqmd().getMessageIdByte();
-                correlationId = l7message.getEnvelope().getMqmd().getCorrelationIdByte();
-
+                MDC.put("messageId", l7message.getEnvelope().getMqmd().getMessageIdByte());
```
Comment (alexjoybc) [inline src/main/java/ca/bc/gov/iamp/bcparis/filter/LoggingFilter.java]: Can you use constants ?

**Review by alexjoybc — CHANGES_REQUESTED** [review]
Add coverage ahahahh

**Review by sdevalapurkar-bcgov — DISMISSED** [review]

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**